openedx/core/lib/tests/test_edx_api_utils.py

```diff
@@ -315,21 +280,10 @@ def test_data_retrieval_failure_with_id(self, mock_exception):
 
         actual = get_edx_api_data(
             catalog_integration,
-            self.user,
             'programs',
             api=api,
             resource_id=100,
             many=False,
```

Comment:
SUPER DUPER NIT:  comma removed from this kwarg above but not here.  Consistency!